src/assets/DatasetsUtils.ts

```diff
@@ -120,7 +120,7 @@ const newDocEntryFactory: { [key: string]: Function } = {
       'name': 'New ACL Profile',
       'description': 'New ACL Profile Description and Remarks',
       'action': 'action-acl-block',
-      'tags': ['all'],
+      'tags': [''],
```

Comment:
```suggestion
      'tags': [],
```
Should be empty array, not array with empty string (otherwise an empty tag will be added to all requests matching this rule)